sys/include/phydat.h

```diff
@@ -157,15 +156,17 @@ void phydat_dump(phydat_t *data, uint8_t dim);
 const char *phydat_unit_to_str(uint8_t unit);
 
 /**
- * @brief   Convert the given scale factor to a NULL terminated string
+ * @brief   Convert the given scale factor to an SI prefix
  *
  * The given scaling factor will be given as SI unit (e.g. M for Mega, u for
- * micro, etc) for obvious cases or in scientific notation (e.g. 2E11, 1E-22,
- * etc) otherwise.
+ * micro, etc), or `\0` otherwise.
```

Comment:
Maybe rephrase a little the detailed descrption:
'The given scaling factor is returned in SI unit ...'

Reply:
addressed